chef-config/lib/chef-config/config.rb

```diff
@@ -379,7 +379,7 @@ def self.derive_path_from_chef_repo_path(child_path)
     #
     # @param path [String]
     def self.path_accessible?(path)
-      File.exist?(path) && File.readable?(path) && File.writable?(path)
+      ChefUtils.file_exist?(path) && File.readable?(path) && File.writable?(path)
```

Comment:
This is only an example of how we are using train helpers `ChefUtiles.file_exist?` which hitting the node to check whether the dir exists or not.

Ref: https://github.com/chef/chef/blob/fb4197f19f60740780c35f4ff37adbd493fdb2dc/chef-utils/lib/chef-utils/dsl/train_helpers.rb#L37-L43